themes/bootstrap3/templates/Recommend/SideFacets/single-facet.phtml

```diff
@@ -34,31 +34,24 @@
   }
 ?>
 
-<?php
-  // Capture badge and exclude link to a variable first:
-  ob_start();
-?>
-<?php if (!$this->facet['isApplied'] && null !== $this->facet['count']): ?>
-    <span class="badge">
-      <?php $currentCount = $this->localizedNumber($this->facet['count']); ?>
-      <span class="sr-only"><?=$this->transEsc('result_count', ['%%count%%' => $currentCount]) ?></span>
-      <span aria-hidden="true"><?=$currentCount ?></span>
+<span class="<?=implode(' ', $classList) ?>">
+  <a class="text<?=$hasCheckbox ? ' icon-link' : ''?>" href="<?=$toggleUrl ?>" data-lightbox-ignore data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>" data-count="<?=$this->facet['count'] ?>"<?php if ($this->facet['isApplied']): ?> title="<?=$this->transEscAttr('applied_filter') ?>"<?php endif;?>>
+    <span class="text">
```

Comment:
Having the class "text" on both the link and this span seems a bit counterintuitive to me.

But with the current styles we need them in both cases and it works. Maybe that is something we can revise in the future.

Reply:
True, but I decided to keep it because it's back-compatible.

Reply:
And yes, we could perhaps revisit the styles in a major release.